client/src/Actions/userSlice.ts

```diff
@@ -0,0 +1,66 @@
+import { createSlice, createAsyncThunk, PayloadAction } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+export type UserType = {
+  id: number;
+  email: string;
+  password: string;
+  name: string;
+  surname: string;
+  avatar: string;
```

Comment:
Avatar i surname są obowiązkowe? Nie powinno być np. avatar?: string ?

Reply:
ogolnie to jest typ ktory przychodzi z backendu a to zawsze przyjdzie niezaleznie czy pole surname jest typu filled czy empty, ale musze jeszcze to zweryfikowac.

Reply:
Okey ale moze nic nie przyjsc z backendu bo bedzie pusto w bazie i wtedy nam sie wysypie typ